Documentation/gettingstarted/local-redirect-policy.rst

```diff
@@ -0,0 +1,279 @@
+
+    kubectl get ciliumlocalredirectpolicies
+    NAME               AGE
+    lrp-metadata       20h
+
+Now deploy a backend pod where the traffic needs to be redirected to. The metadata
+labels and container port and protocol match with the labels, port and protocol
+fields specified in the CiliumLocalRedirectPolicy custom resource created above.
+
+.. code-block:: yaml
+
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: lrp-pod
+      labels:
+        app: proxy
+    spec:
+      containers:
+      - name: lrp-pod
+        image: nginx
+        ports:
+        - containerPort: 80
+          protocol: TCP
+
+Verify that the pod is running.
+
+.. parsed-literal::
+
+    kubectl get pods | grep lrp-pod
+    lrp-pod                      1/1     Running   0          46s
+
+Deploy a client pod that will invoke a curl command to the IP address and port
+configuration specified in the CiliumLocalRedirectPolicy CR above in order to match
+traffic that needs to be redirected.
+
+.. parsed-literal::
+
+    kubectl exec nginx-client -- curl -I -s http://169.254.169.254:8080/index.html
+    HTTP/1.1 200 OK
+    Server: nginx/1.19.2
+    Date: Fri, 28 Aug 2020 01:33:34 GMT
+    Content-Type: text/html
+    Content-Length: 612
+    Last-Modified: Tue, 11 Aug 2020 14:50:35 GMT
+    Connection: keep-alive
+    ETag: "5f32b03b-264"
+    Accept-Ranges: bytes
+
+Verify that the traffic was redirected to the ``lrp-pod`` that was deployed.
+
+.. parsed-literal::
+
+    kubectl describe pod lrp-pod  | grep 'IP:'
+    IP:           10.16.70.187
+
+.. parsed-literal::
+
+    sudo tcpdump -i any -n port 80
+    tcpdump: verbose output suppressed, use -v or -vv for full protocol decode
+    listening on any, link-type LINUX_SLL (Linux cooked), capture size 262144 bytes
+    01:36:24.608566 IP 10.16.215.55.60876 > 10.16.70.187.80: Flags [S], seq 2119454273, win 28200, options [mss 1410,sackOK,TS val 2541637677 ecr 0,nop,wscale 7], length 0
+    01:36:24.608600 IP 10.16.70.187.80 > 10.16.215.55.60876: Flags [S.], seq 1315636594, ack 2119454274, win 27960, options [mss 1410,sackOK,TS val 2962246962 ecr 2541637677,nop,wscale 7], length 0
+    01:36:24.608638 IP 10.16.215.55.60876 > 10.16.70.187.80: Flags [.], ack 1, win 221, options [nop,nop,TS val 2541637677 ecr 2962246962], length 0
+    01:36:24.608867 IP 10.16.215.55.60876 > 10.16.70.187.80: Flags [P.], seq 1:96, ack 1, win 221, options [nop,nop,TS val 2541637677 ecr 2962246962], length 95: HTTP: HEAD /index.html HTTP/1.1
+    01:36:24.608876 IP 10.16.70.187.80 > 10.16.215.55.60876: Flags [.], ack 96, win 219, options [nop,nop,TS val 2962246962 ecr 2541637677], length 0
+    01:36:24.609007 IP 10.16.70.187.80 > 10.16.215.55.60876: Flags [P.], seq 1:239, ack 96, win 219, options [nop,nop,TS val 2962246962 ecr 2541637677], length 238: HTTP: HTTP/1.1 200 OK
+    01:36:24.609052 IP 10.16.215.55.60876 > 10.16.70.187.80: Flags [.], ack 239, win 229, options [nop,nop,TS val 2541637677 ecr 2962246962], length 0
+
```

Comment:
I see a value in having this verification step since it confirms that 1) the frontend traffic was indeed redirected 2) the traffic was redirected to the backend pod selected by the LRP.